### PR TITLE
Release 2.8.6

### PR DESCRIPTION
**Pull request by oscarssanchezz**
### Description of the Change

PR Release for v.2.8.6

### Credits

N/A

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ All notable changes to this project will be documented in this file, per [the Ke
 ### Security
 -->
 
+## [2.8.6] - 2023-05-07
```
Comment (felipeelia) [inline CHANGELOG.md:16]:
```suggestion
## [2.8.6] - 2024-05-07
```

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "brightcove-video-connect",
-  "version": "2.8.5",
+  "version": "2.8.6",
```
Comment (felipeelia) [inline package-lock.json:3]: this also needs to be changed in line 9

File: readme.txt [hunk 3]
```diff
@@ -39,6 +39,13 @@ For installation, usage, and Frequently Asked Question please see the [Brightcov
 
 == Changelog ==
 
+= 2.8.6 - 2023-05-07 =
+
+__Fixed:__
+
+- Cannot use Brightcove block on custom post types. Props [@spacedmonkey](https://github.com/spacedmonkey) and [@jonnynews](https://github.com/jonnynews).
+- Typo when saving the language_detection attribute. Props [@claimableperc](https://github.com/claimableperc).
```
Comment (felipeelia) [inline readme.txt:47]: Can we make this list to start with `*` to line up with the previous ones?

**Review by felipeelia — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (oscarssanchezz) [reply CHANGELOG.md]: Fixed here: https://github.com/10up/brightcove-video-connect/pull/391/commits/dfba57ce996ef65a87983b7a4c40e977ae96c804#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL16

File: package-lock.json (hunk 2 above)
Comment (oscarssanchezz) [reply package-lock.json:3]: Fixed here https://github.com/10up/brightcove-video-connect/pull/391/commits/dfba57ce996ef65a87983b7a4c40e977ae96c804#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R9

File: readme.txt (hunk 3 above)
Comment (oscarssanchezz) [reply readme.txt]: Fixed here https://github.com/10up/brightcove-video-connect/pull/391/commits/dfba57ce996ef65a87983b7a4c40e977ae96c804#diff-9e6e4772050998a5c0dc3c61acf3dab0a7e594566171fa5746d6b62f9598efb6R46

File: CHANGELOG.md [hunk 4]
```diff
@@ -13,7 +13,15 @@ All notable changes to this project will be documented in this file, per [the Ke
...
 -->
 
-## [2.8.5] - 2023-04-22
+## [2.8.6] - 2024-05-08
+
+### Fixed
+
+- Cannot use Brightcove block on custom post types. Props [@spacedmonkey](https://github.com/spacedmonkey) and [@jonnynews](https://github.com/jonnynews) via [#389](https://github.com/10up/brightcove-video-connect/pull/389).
+- Typo when saving the language_detection attribute. Props [@claimableperc](https://github.com/claimableperc) via [#390](https://github.com/10up/brightcove-video-connect/pull/390).
+- Bugfix: combined queries with q search parameter. Props [@oscarssanchezz](https://github.com/oscarssanchezz/oscarssanchezz) via [#392](https://github.com/10up/brightcove-video-connect/pull/392).
```
Comment (felipeelia) [inline CHANGELOG.md:22]:
```suggestion
-Combined queries with q search parameter. Props [@oscarssanchezz](https://github.com/oscarssanchezz/oscarssanchezz) via [#392](https://github.com/10up/brightcove-video-connect/pull/392).
```
(as it is already in the "Fixed" section, we don't need the "Bugfix" prefix ;) )

**Review by felipeelia — APPROVED** [review]

**Pull request merged**